sql/SQLCode.sql

```diff
@@ -0,0 +1,26 @@
+CREATE User(
+user_ID VARCHAR(5),
```

Comment:
```suggestion
user_id INT AUTO_INCREMENT PRIMARY KEY,
```

Reply:
This should be applied to every id. And please also change the foreignkeys datatypes to match